package.json

```diff
@@ -32,21 +40,16 @@
     "d3-scale": "^3.0.0",
     "d3-selection": "^1.4.0",
     "d3-shape": "^1.3.4",
-    "del": "^4.1.1",
+    "del-cli": "^2.0.0",
     "eslint": "^5.14.1",
-    "gulp": "^4.0.2",
-    "gulp-rename": "^1.4.0",
-    "gulp-sass": "^4.0.2",
-    "node-sass": "^4.12.0",
     "prettier": "^1.16.4",
-    "rollup": "^1.14.0",
+    "rollup": "^1.15.6",
     "rollup-plugin-commonjs": "^10.0.0",
-    "rollup-plugin-node-resolve": "^5.0.1",
+    "rollup-plugin-node-resolve": "^5.0.3",
+    "rollup-plugin-postcss": "^2.0.3",
     "rollup-plugin-uglify": "^6.0.2",
-    "typeface-oswald": "0.0.54",
-    "vinyl-buffer": "^1.0.1",
-    "vinyl-source-stream": "^2.0.0",
-    "yargs": "^13.2.4"
+    "sass": "^1.21.0",
```

Comment:
:+1: i actually don't know they have js version.

Reply:
Actually it is written in dart but transpiled back to js. It is now official sass implementation. 😎